framework/caching/Cache.php

```diff
@@ -101,8 +101,11 @@ public function buildKey($key)
      */
     public function get($key)
     {
-        $key = $this->buildKey($key);
-        $value = $this->getValue($key);
+        \Yii::beginProfile($key);
```

Comment:
Need to check how these calls affect production performance. If much, add a check for `YII_DEBUG`.